### PR TITLE
Adding compatibility with python 3

### LINKED ISSUE #39: Make python 3 compatible
Should convert code to be python 3 compatible.

### PR DESCRIPTION
**Pull request by bhazelton**
closes #39

### CODE REVIEW

**Comment by mkolopanis** [comment]
This closes #39 but does that need to be in the top level comment?

**Comment by bhazelton** [comment]
yes, the closing text needs to be in the description, I'll add it.

File: pyuvsim/tests/test_mpi_uvsim.py [hunk 1]
```diff
@@ -62,8 +64,16 @@ def test_run_param_uvsim():
     uv_new = UVData()
     uv_new.read_uvfits(tempfilename)
     os.remove(tempfilename)
+
+    time_obj = time.Time(uv_new.time_array[0], format='jd')
+    uv_new.unphase_to_drift(use_ant_pos=True)
+    uv_new.phase_to_time(time_obj)
```
Comment (bhazelton) [inline pyuvsim/tests/test_mpi_uvsim.py]: On my machine, I don't need this `phase_to_time(time_obj)` line, the objects match with just the `unphase_to_drift(use_ant_pos=True)` added. What was the problem you were seeing without the `phase_to_time`?

Comment (bhazelton) [reply pyuvsim/tests/test_mpi_uvsim.py]: I see it. It's actually that the uv_new object in the lower block of code (after the votable stuff)  is entirely new -- read from a uvfits file (so it's a phased object), while it's being compared to the uv_ref that was already unphased. Adding `uv_new.unphase_to_drift(use_ant_pos=True)` before the final assert fixes it.

Comment (mkolopanis) [reply pyuvsim/tests/test_mpi_uvsim.py]: ah I missed that second block. I will add that change.

**Comment by dannyjacobs** [comment]
Adds python3 compatibility
Fixes coverall incorrectly reports coverage decreasing

**Pull request merged**